src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+  body {
+  margin: 0 auto;
+  font-family: Roboto, sans-serif;
+  }
+
+  .header {
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  background-color: #fff;
+  display: flex;
+  justify-content:space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding for your links instead to make a height you need